docs/topics/feed-exports.rst

```diff
@@ -429,3 +430,40 @@ format in :setting:`FEED_EXPORTERS`. E.g., to disable the built-in CSV exporter
 .. _Amazon S3: https://aws.amazon.com/s3/
 .. _botocore: https://github.com/boto/botocore
 .. _Canned ACL: https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl
+
+.. setting:: FEED_STORAGE_BATCH_ITEM_COUNT
+
+FEED_STORAGE_BATCH_ITEM_COUNT
+-----------------------------
+Default: ``None``
+
+An integer number that represents the number of scraped items stored in each output
+file. Whenever the number of items exceeds this setting, a new file is
+created and the output is redirected to it.  The name of the new file will be selected
+based on the timestamp when the feed is being created and/or on the batch sequence number.
+Therefore you must specify %(batch_time)s or %(batch_id)s or both in :setting:`FEED_URI`.
```

Comment:
Some things to consider:
- It should be more obvious, and explained as early as possible, that Scrapy only generates multiple output files if this set to a different value (so, not by default).
- This paragraph covers two significantly different topics (the setting value, and the handling of file names when a non-zero value is used), so different paragraphs make sense.

For example:
```suggestion
If assigned an integer number higher than `0`, Scrapy generates multiple output files
storing up to the specified number of items in each output file.

When generating multiple output files, you must use at least one of the following
placeholders in :setting:`FEED_URI` to indicate how the different output file names are
generated:
```

Reply:
> If assigned an integer number higher than `0`

Should we also use double quotes here?

Reply:
Indeed :facepalm:

Reply:
Also, mind that to make a separate paragraph a new line is not enough. You need an empty line in between.